pkg/auth/auth_handler_test.go

```diff
@@ -186,6 +186,10 @@ func (f fakeInventory) DisableHost(ctx context.Context, params installer.Disable
 	panic("Implement Me!")
 }
 
+func (f fakeInventory) GetPresignedForClusterFiles(ctx context.Context, params installer.GetPresignedForClusterFilesParams) middleware.Responder {
+	panic("Implement Me!")
```

Comment:
its that intentional?

Reply:
That's the pattern here, I just followed it...